README.md

```diff
@@ -11,58 +11,29 @@
 
 <!-- <<<Super-quickstart links go here>>> -->
 
-
-
 <img align="right" src="./docs/docusaurus/static/img/gx-mark-160.png">
 
-Great Expectations
```

Comment:
I think saying something about 1.0 and linking to the blog post is still a good idea

Reply:
@joshzzheng I'm not sure if your feedback was rhetorical, but the revised README contains a section about the 1.0 release and a link to the blog post: 

![image](https://github.com/great-expectations/great_expectations/assets/104205257/10451354-a6fe-416e-a706-59ac349a285f)

Reply:
Yup I saw it further down the page. lgtm.